utils/node_local_test.py

```diff
@@ -16,14 +16,20 @@
 
 from collections import OrderedDict
 
-class DFTestFail(Exception):
+class NLTestFail(Exception):
     """Used to indicate test failure"""
     pass
 
-class DFTestNoFi(DFTestFail):
+class NLTestNoFi(NLTestFail):
     """Used to indicate Fault injection didn't work"""
     pass
 
+class NLTestNoFunction(NLTestFail):
+    """Used to indicate a function did not log anything"""
+
+    def __init__(self, function):
```

Comment:
(pylint-super-init-not-called) __init__ method from base class 'NLTestFail' is not called